### DISCUSSION TITLE
How to use the Select component in “tags” mode without rendering the dropdown?

### DISCUSSION THREAD

**Discussion by nikzanda** [ant-design/ant-design#50173] [Q&A]
I would like an input where I can type multiple words (tags) without rendering the dropdown, without search filters, etc. I just want to be able to write, press “enter,” and see the tag with the written word, and so on. Could this be an idea for a new dedicated component?
![Screenshot 2024-07-31 at 11 10 15](https://github.com/user-attachments/assets/cce8f69d-5bd9-45fb-8515-1e1a63980ac2)

**Comment by ByYogi** [comment]
I think it's possible to solve your problem temporarily with `dropdownRender` first
``` tsx
<Select
    mode="tags"
    style={{ width: '100%' }}
    placeholder="Tags Mode"
    onChange={handleChange}
    options={options}
dropdownRender={() => <>Type tags</>}
  />
```

**Reply by nikzanda** [reply]
Ok, thanks. So, at the moment, there is no way to avoid rendering the dropdown. I will open a feature request.

**Comment by nikzanda** [comment]
The solution: https://github.com/ant-design/ant-design/issues/50210#issuecomment-2265441276

**Reply by ByYogi** [reply]
cool
